test/recipes/30-test_evp_data/evpdigest.txt

```diff
@@ -167,6 +167,62 @@ Input = "a"
 Ncopy = 64
 Output = e718483d0ce769644e2e42c7bc15b4638e1f98b13b2044285632a803afa973ebde0ff244877ea60a4cb0432ce577c31beb009c5c2c49aa2e4eadb217ad8cc09b
 
+Digest = SHA512-224
+Input = 
+Output = 6ed0dd02806fa89e25de060c19d3ac86cabb87d6a0ddd05c333b84f4
```

Comment:
Reference to source would be more than appropriate. I mean where the test vectors come from.

Reply:
The vectors are from NIST, I'll add references tomorrow.